integration/abi/AugurEventEmitter.abi.json

```diff
@@ -0,0 +1 @@
type":"uint256"},{"indexed":false,"name":"payoutNumerators","type":"uint256[]"}],"name":"ReportSubmitted","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"reporter","type":"address"},{"indexed":true,"name":"market","type":"address"},{"indexed":false,"name":"reportingToken","type":"address"},{"indexed":false,"name":"amountRedeemed","type":"uint256"},{"indexed":false,"name":"reportingFeesReceived","type":"address"},{"indexed":false,"name":"payoutNumerators","type":"uint256[]"}],"name":"WinningTokensRedeemed","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"disputer","type":"address"},{"indexed":true,"name":"market","type":"address"},{"indexed":false,"name":"reportingPhase","type":"uint8"},{"indexed":false,"name":"disputeBondAmount","type":"uint256"}],"name":"ReportsDisputed","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"market","type":"address"}],"name":"MarketFinalized","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"universe","type":"address"}],"name":"UniverseForked","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"market","type":"address"},{"indexed":true,"name":"sender","type":"address"},{"indexed":false,"name":"outcome","type":"uint8"},{"indexed":false,"name":"orderId","type":"bytes32"},{"indexed":false,"name":"orderType","type":"uint8"},{"indexed":false,"name":"tokenRefund","type":"uint256"},{"indexed":false,"name":"sharesRefund","type":"uint256"}],"name":"OrderCanceled","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"shareToken","type":"address"},{"indexed":true,"name":"creator","type":"address"},{"indexed":true,"name":"orderId","type":"bytes32"},{"indexed":false,"name":"price","type":"int256"},{"indexed":false,"name":"amount","type":"uint256"},{"indexed":false,"name":"numTokensEscrowed","type":"uint256"},{"indexed":false,"name":"numSharesEscrowed","type":"uint256"},{"indexed":false,"name":"tradeGroupId","type":"bytes32"}],"name":"OrderCreated","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"shareToken","type":"address"},{"indexed":true,"name":"creator","type":"address"},{"indexed":true,"name":"filler","type":"address"},{"indexed":false,"name":"price","type":"int256"},{"indexed":false,"name":"numCreatorShares","type":"uint256"},{"indexed":false,"name":"numCreatorTokens","type":"uint256"},{"indexed":false,"name":"numFillerShares","type":"uint256"},{"indexed":false,"name":"numFillerTokens","type":"uint256"},{"indexed":false,"name":"settlementFees","type":"uint256"},{"indexed":false,"name":"tradeGroupId","type":"bytes32"}],"name":"OrderFilled","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"sender","type":"address"},{"indexed":true,"name":"market","type":"address"},{"indexed":false,"name":"numShares","type":"uint256"},{"indexed":false,"name":"numPayoutTokens","type":"uint256"},{"indexed":false,"name":"finalTokenBalance","type":"uint256"}],"name":"ProceedsClaimed","type":"event"}]
```

Comment:
Recommend not committing derived files to the repo.  Assuming this is the standard contract ABI, this can be generated with `solc` pretty trivially.  Consider instead committing the code necessary to generate this file?  e.g., npm dependency on `solc` plus a little TS script that compiles the attached file and generates standard-json output file?

Reply:
Sure, I was copying what i found in augur-contracts:

https://github.com/AugurProject/augur-contracts/tree/master/abi

i agree

Reply:
I don't think it's a bad idea to commit the ABI JSON file to the repo.  AFAIK most projects do this -- e.g. [0x](https://github.com/0xProject/contracts/tree/master/build/contracts), [gnosis](https://github.com/gnosis/gnosis-contracts/blob/master/networks.json), etc.

Reply:
OK, that's simpler for sure. let's go with that, for now at least unless this gets more complicated, which i don't expect.